### PR TITLE
Adding events 2022-11-16

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2022-11-16-this-week-in-rust.md [hunk 1]
```diff
@@ -117,51 +101,78 @@ Rusty Events between 2022-11-16 - 2022-12-14 🦀
...
+    * [**Rust Hack and Learn**](https://berline.rs/2022/12/06/rust-hack-and-learn.html)
 * 2022-12-06 | Virtual (Buffalo, NY, US) | [Buffalo Rust Meetup](https://www.meetup.com/buffalo-rust-meetup/)
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/buffalo-rust-meetup/events/hlgvxsydcqbjb/)
 * 2022-12-07 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/287027660/)
+* 2022-12-08 | Virtual (Nürnberg, DE) | [Rust Nuremberg](https://www.meetup.com/rust-noris/)
+    * [**Rust Nürnberg online #20**](https://www.meetup.com/rust-noris/events/hlvbvsydcqblb/)
+* 2022-12-10 | Virtual | [Rust GameDev](https://gamedev.rs/)
+    * [**Rust GameDev Monthly Meetup**](https://www.google.com/url?q=https%3A%2F%2Fdiscord.gg%2FyNtPTb2&sa=D&ust=1666661760000000&usg=AOvVaw13uHY9m-8bJJwgeP58VS8l)
+* 2022-12-13 | Virtual (Saarbrücken, DE) | [Rust-Saar](https://www.meetup.com/rust-saar/)
```
Comment (mariannegoldin) [inline draft/2022-11-16-this-week-in-rust.md:124]: Can we consider adding for 12/13: https://www.meetup.com/altow-academy/events/289352426/

File: draft/2022-11-16-this-week-in-rust.md [hunk 2]
```diff
@@ -89,22 +89,6 @@ Rusty Events between 2022-11-16 - 2022-12-14 🦀
...
-    * [**Rust Nürnberg online #19**](https://www.meetup.com/rust-noris/events/hlvbvsydcpbnb/)
-* 2022-11-12 | Virtual | [Rust GameDev](https://gamedev.rs/)
-    * [**Rust GameDev Monthly Meetup**](https://www.google.com/url?q=https%3A%2F%2Fdiscord.gg%2FyNtPTb2&sa=D&ust=1666661760000000&usg=AOvVaw13uHY9m-8bJJwgeP58VS8l)
-* 2022-11-15 | Virtual (Nairobi, KE / New York, NY, US)| [Data Umbrella Africa](https://www.meetup.com/data-umbrella-africa2/)
-    * [**Online: Introduction to Rust Programming**](https://www.meetup.com/data-umbrella-africa2/events/289308825/) | [**New York Mirror**](https://www.meetup.com/data-umbrella/events/289308172/)
-* 2022-11-15 | Virtual (Washington, DC, US) | [Rust DC](https://www.meetup.com/rustdc//)
-    * [**Mid-month Rustful**](https://www.meetup.com/rustdc/events/289015883/)
 * 2022-11-16 | Virtual (Vancouver, BC, CA) | [Vancouver Rust](https://www.meetup.com/vancouver-rust)
     * [**Rust Study/Hack/Hang-out**](https://www.meetup.com/vancouver-rust/events/tqvhxsydcpbvb/)
 * 2022-11-17 | Virtual (Amsterdam, NL) | [ITGilde Tech-Talks](https://www.meetup.com/itgilde-cooperatie-amsterdam-unix-linux-meetups)
```
Comment (mariannegoldin) [inline draft/2022-11-16-this-week-in-rust.md:94]: Would we consider this event for 11/17? The tool was built in Rust, and it sounds like the talk is about using Rust and Python together in one project: https://www.meetup.com/data-ai-online/events/289623322/

File: draft/2022-11-16-this-week-in-rust.md [hunk 3]
```diff
@@ -117,51 +101,78 @@ Rusty Events between 2022-11-16 - 2022-12-14 🦀
...
     * [**Rust Munich 2022 / 3 - hybrid**](https://www.meetup.com/rust-munich/events/289065390/)
 * 2022-12-01 | Virtual (Charlottesville, VA, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
     * [**Exploring USB with Rust**](https://www.meetup.com/charlottesville-rust-meetup/events/289563986/)
+* 2022-12-06 | Virtual (Berlin, DE) | [Berlin.rs](https://berline.rs/)
+    * [**Rust Hack and Learn**](https://berline.rs/2022/12/06/rust-hack-and-learn.html)
 * 2022-12-06 | Virtual (Buffalo, NY, US) | [Buffalo Rust Meetup](https://www.meetup.com/buffalo-rust-meetup/)
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/buffalo-rust-meetup/events/hlgvxsydcqbjb/)
 * 2022-12-07 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/287027660/)
+* 2022-12-08 | Virtual (Nürnberg, DE) | [Rust Nuremberg](https://www.meetup.com/rust-noris/)
```
Comment (mariannegoldin) [inline draft/2022-11-16-this-week-in-rust.md:120]: Related to the comment [above](https://github.com/rust-lang/this-week-in-rust/pull/3812/files#r1024337314), would we consider this event for 12/8? The related project was built in Rust, and the meetup is about "why Rust and (tool)" - https://www.meetup.com/data-ai-online/events/289672886/

File: draft/2022-11-16-this-week-in-rust.md (hunk 2 above)
Comment (mariannegoldin) [inline draft/2022-11-16-this-week-in-rust.md:94]: Recommending to add this event on 11/17, (Rust + Wasm with Alex Crichton): https://www.meetup.com/wasmna/events/289623268/

File: draft/2022-11-16-this-week-in-rust.md (hunk 1 above)
Comment (andrewpollack) [reply draft/2022-11-16-this-week-in-rust.md:124]: Added

File: draft/2022-11-16-this-week-in-rust.md (hunk 2 above)
Comment (andrewpollack) [reply draft/2022-11-16-this-week-in-rust.md:94]: Added

Comment (andrewpollack) [reply draft/2022-11-16-this-week-in-rust.md:94]: Added

**Comment by andrewpollack** [comment]
Excellent suggestions! Added all, and committing/merging now

File: draft/2022-11-16-this-week-in-rust.md (hunk 3 above)
Comment (andrewpollack) [reply draft/2022-11-16-this-week-in-rust.md:124]: Missed this comment -- addressed in https://github.com/andrewpollack/this-week-in-rust/pull/new/sup-2022-11-16

**Pull request merged**